index.js

```diff
@@ -53,6 +53,9 @@ app.use("/message", messageRoutes);
 // Swagger routes
 app.use('/docs', swaggerUi.serve, swaggerUi.setup(swaggerDocument))
 
+// Health check for GCP stuff
+app.use("/health", (req, res) => { res.status(200); } );
```

Comment:
i think it should be `res.sendStatus(200)`

Reply:
So it should be `app.use("/health", res.sendStatus(200));`?

Reply:
`app.use("/health", (req, res) => res.sendStatus(200);`